Docker image unable to build

Build log throws this error:
```
go: finding gopkg.in/jcmturner/rpc.v1 v1.1.0
# github.com/RedHatInsights/insights-results-aggregator/server
server/errors.go:67:32: responses.SendError(writer, err.Error()) used as value
server/errors.go:69:32: responses.SendError(writer, err.Error()) used as value
server/errors.go:71:13: undefined: responses.SendNotFound
server/errors.go:73:36: responses.SendForbidden(writer, err.Error()) used as value
server/errors.go:75:46: responses.SendInternalServerError(writer, "Internal Server Error") used as value
server/server.go:97:31: responses.SendResponse(writer, responses.BuildOkResponse()) used as value
server/server.go:110:30: responses.SendResponse(writer, responses.BuildOkResponseWithData("organizations", organizations)) used as value
server/server.go:130:30: responses.SendResponse(writer, responses.BuildOkResponseWithData("clusters", clusters)) used as value
server/server.go:211:30: responses.SendResponse(writer, responses.BuildOkResponseWithData("report", response)) used as value
server/server.go:295:30: responses.SendResponse(writer, responses.BuildOkResponse()) used as value
server/server.go:295:30: too many errors
make: *** [Makefile:11: build] Error 2
```

--- a/go.mod
+++ b/go.mod
@@ -5,11 +5,12 @@ go 1.13
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/DATA-DOG/go-sqlmock v1.4.1
-	github.com/RedHatInsights/insights-operator-utils v0.0.0-20200212101518-5819e1fc31a9
+	github.com/RedHatInsights/insights-operator-utils v0.0.0-20200323131325-73813484635c
 	github.com/Shopify/sarama v1.26.0
 	github.com/bitly/go-simplejson v0.5.0 // indirect
 	github.com/deckarep/golang-set v1.7.1
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/droptheplot/abcgo v0.0.0-20171120220436-23529565504c // indirect
 	github.com/gchaincl/sqlhooks v1.3.0
 	github.com/go-yaml/yaml v2.1.0+incompatible
 	github.com/google/uuid v1.1.1

